syntax = "proto3";
import "google/protobuf/any.proto";

package visualfl;

message Task {
  string job_id = 1;
  string task_id = 2;
  string web_task_id = 3;
  string task_type = 4;
  google.protobuf.Any task = 5;
  string assignee = 6;
}

